Taskfile.yml

```diff
@@ -360,6 +361,11 @@ tasks:
         docker compose run --rm legacy-mongo-shell
         'mongodb://username:password@host.docker.internal:27017/'
 
+  mcpclient:
+    desc: "Run MCP client (using local ollama server)"
```

Comment:
It is not correct to call it "host": https://github.com/mark3labs/mcphost?tab=readme-ov-file#overview-

Reply:
Thank you :pray: